app/staging/domain/case-details.js

```diff
@@ -1,12 +1,13 @@
 class CaseDetails {
-  constructor (rowType, caseRefNo, tierCode, teamCode, omGradeCode, omKey, location) {
+  constructor (rowType, caseRefNo, tierCode, teamCode, omGradeCode, omKey, location, workloadId) {
     this.rowType = rowType
     this.caseRefNo = caseRefNo
     this.tierCode = tierCode
     this.teamCode = teamCode
     this.omGradeCode = omGradeCode
     this.omKey = omKey
     this.location = location
+    this.workloadId = workloadId
```

Comment:
The staging schema doesn't have a workload table and the app workload won't have been created yet so the workloadId won't exist when we create any staging domain object.